indico/modules/rb_new/blueprint.py

```diff
@@ -39,6 +39,8 @@
 _bp.add_url_rule('/api/map-areas', 'map_areas', misc.RHMapAreas)
 _bp.add_url_rule('/api/equipment', 'equipment_types', misc.RHEquipmentTypes)
 _bp.add_url_rule('/api/locations', 'locations', locations.RHLocations)
+_bp.add_url_rule('/api/data/<any(event,contrib,session_block):type>/<int:id>', 'linked_object_data',
```

Comment:
`/api/link-info/...`? "data" is way too generic..

also, i'd `s/contrib/contribution/`